tests/rbd/rbd_utils.py

```diff
@@ -111,8 +111,9 @@ def exec_cmd(self, **kw):
             self.flag = 1
             return 1
 
-    def random_string(self):
-        temp_str = "".join([random.choice(string.ascii_letters) for _ in range(10)])
+    def random_string(self, **kw):
+        size = kw.get("size", 10)
```

Comment:
can we make it as `string_size` instead of the generic `size` it will bring confuse to image size.

Reply:
Changed it to len instead of size, to signify string length